app/bundles/CoreBundle/EventListener/BuildJsSubscriber.php

```diff
@@ -137,6 +137,30 @@ public function onBuildJs(BuildJsEvent $event)
     return response;
 };
 
+MauticJS.isPixelLoaded = function() {
+    if (typeof MauticJS.trackingPixel !== 'undefined') {
+        return /co/.test(MauticJS.trackingPixel);
```

Comment:
Gotcha. Thanks! It returned true so matching something. Probably the URL if that's the case. Haha.  I'll try the .complete state although that doesn't seem to work for the dynamically loaded pixel.